adafruit_magtag/magtag.py

```diff
@@ -65,16 +70,16 @@ class MagTag(PortalBase):
     def __init__(
         self,
         *,
-        url=None,
-        headers=None,
-        json_path=None,
-        regexp_path=None,
-        default_bg=0xFFFFFF,
-        status_neopixel=None,
-        json_transform=None,
-        rotation=270,
-        debug=False,
-    ):
+        url: Optional[str] = None,
+        headers: Optional[Dict[str, str]] = None,
+        json_path: Optional[Sequence[Any]] = None,
+        regexp_path: Optional[Sequence[str]] = None,
+        default_bg: Union[str, int] = 0xFFFFFF,
+        status_neopixel: Optional[str] = None,
```

Comment:
I think `status_neopixel` should be `Optional[microcontroller.Pin]`

The import for microcontroller can be added inside the try/except block after the typing import.

Reply:
Oh, sorry, missed that one!